cmd/agent/app/start.go

```diff
@@ -234,12 +230,19 @@ func StartAgent() error {
 
 // setupMetadataCollection initializes the metadata scheduler and its collectors based on the config
 func setupMetadataCollection(s *serializer.Serializer, hostname string) error {
```

Comment:
What exactly?  Function insides?

Because signature is the same  
![image](https://user-images.githubusercontent.com/701233/56962740-93f2cb00-6b57-11e9-9352-24b21c6dbb3f.png)

Reply:
yes we changed the behavior of `setupMetadataCollection` not the signature, so retain the function as on master